### ISSUE TITLE
spotify not playing 

### ISSUE THREAD

**Issue by leonx132** [dlrudie/Snip#260]
spotify is playing but snip isn't recognizing that, i can seem to get it to work for all of a second when i 1st start them both up then it just goes back to that bug

**Comment by KiriJolith** [comment]
I'm having the same issue. Worked just fine when I first did it, but after about 20 minutes it now only works for a split second after it opens.

**Comment by mov51** [comment]
having the same issue, I came to this program after snaz stopped supporting now playing, seems snaz was right in their not supporting spotify anymore.

**Comment by Lygolp** [comment]
Same Problem! "No Track playing at the Moment"  but Spotify play!

**Comment by MissPotato** [comment]
If I had to take a guess, a piece of the API probably changed that @dlrudie uses tor Snip.

**Comment by dlrudie** [comment]
Been busy with work. It's probably been a few months or more since I really had a chance to sit down with Snip. I just fired it up and made sure Spotify is the latest version. Snip is still working fine here. Make sure SpotifyWebHelper is enabled and running. That's where Snip gets track information from.

**Comment by MissPotato** [comment]
Ah yes! That's it, I ave SWH disabled on startup, I'll probably add that to streaming script, thanks Dlrudie!

**Comment by MissPotato** [comment]
So I deleted the first one because my issue started happening again. I started web helper manually and it didn't work, then I turned on the boot with windows, and it still didn't work. So then I uninstalled Snip and installed the latest version, and that fixed it. So, if you're also running <=5.6.5 you probably experiencing the bug too. Makes me wonder how long my stream has been w/o song credits...

TL:DR Update snip, you're liking running an outdated version!
